Listenbrainz/UI/Screens/ContentView.swift

```diff
@@ -1,53 +1,45 @@
-//
-//  ContentView.swift
-//  Listenbrainz
-//
-//  Created by Gaurav Bhardwaj on 16/04/23.
-//
-
+// ContentView.swift
 import SwiftUI
 import UIKit
 
 struct ContentView: View {
-  @State private var userName:String = ""
-  let screenWidth = UIScreen.main.bounds.size.width
-  let screenHeight = UIScreen.main.bounds.size.height
-  @EnvironmentObject var homeViewModel: HomeViewModel
-  @EnvironmentObject var feedViewModel: FeedViewModel
-
-
-  init() {
-    UITabBar.appearance().layer.borderColor = UIColor.clear.cgColor
-    UITabBar.appearance().clipsToBounds = true
-  }
-
-  var body: some View {
-    ZStack{
-      TabView {
-
-        FeedView()
-          .environmentObject(feedViewModel)
-          .frame(width: screenWidth, height: screenHeight, alignment: .center)
-          .tabItem {
-            Label("Feed", systemImage: "bolt")
-          }
-        PlayerView()
-          .edgesIgnoringSafeArea(.all)
-          .frame(width: screenWidth, height: screenHeight, alignment: .center)
-          .tabItem {
-            Label("Player", systemImage: "opticaldisc.fill")
-          }
-
-        ListensView()
-          .environmentObject(homeViewModel)
-          .frame(width: screenWidth, height: screenHeight, alignment: .center)
-          .tabItem {
-            Label("Listens", systemImage: "person.wave.2")
-          }
+    @State private var hasCompletedOnboarding: Bool = false
+    @State private var userName: String = ""
+    let screenWidth = UIScreen.main.bounds.size.width
```

Comment:
You shouldn't need to do this. Rethink it

Reply:
:( done